src/components/footer/footer.module.css

```diff
@@ -1,18 +1,28 @@
-@custom-media --list-horizontally (min-width: 1400px);
+@custom-media --list-horizontally (min-width: 1280px);
 
 .root {
+	--padding: 24px;
+	--logo-button-margin-bottom: 8px;
+
 	align-items: center;
 	color: var(--token-color-palette-neutral-600);
 	display: flex;
 	flex-wrap: wrap;
 	justify-content: space-between;
-	padding: 32px 0;
+	padding: var(--padding);
 	position: relative;
 }
 
 .logo {
-	margin: 8px 40px 8px 0;
+	margin: 0 24px var(--logo-button-margin-bottom) 0;
 	position: relative;
+	border-radius: 6px;
+
+	&:focus,
+	&:focus-visible {
+		box-shadow: var(--token-focus-ring-action-box-shadow);
+		outline: transparent;
+	}
```

Comment:
I think what you were looking for is the `g-focus-ring-from-box-shadow` helper. 😊

Can you replace these lines with a `composes` call at the top of the `.logo` block?

```
composes: g-focus-ring-from-box-shadow from global;
```

Reply:
Thank you! that is the helper I need 😆 much better 👐🏻